test/reduce.jl

```diff
@@ -16,6 +17,7 @@
 @test Base.mapfoldl((x)-> x ⊻ true, |, [true false true false false]) == true
 @test Base.mapfoldl((x)-> x ⊻ true, |, false, [true false true false false]) == true
 
+@test foldr(+, Int64[]) == 0
```

Comment:
Referencing via the issue / pull request numbers (e.g. #20144, #20160) would be great. Given #20160 adds the `foldl` test and #20144/#20160 contain the relevant discussion, perhaps reference #20144 and #20160 for that test as well, rather than (or in addition to) #7465? Thanks!